Develop an SQL schema for SOPs

```diff
@@ -342,106 +480,203 @@ CREATE SEQUENCE public."PROTOCOLS_Protocol_ID_seq"
+    "instrument-detector" text,
+    software_number integer NOT NULL,
+    software text NOT NULL,
+    setting_number integer,
+    "software-setting" text,
+    publication_number integer,
+    publication text,
+    "contact-name_number" integer,
+    "contact-name" text,
+    "contact-affiliation_number" integer,
+    "contact-affiliation" text,
+    "contact-email_number" integer,
+    "contact-email" text,
+    uri_number integer,
+    uri text,
+    external_study_uri_number integer,
+    external_study_uri text,
+    quantification_method text NOT NULL,
+    sample_number integer,
+    sample text,
+    species_number integer,
+    "sample-species" text,
+    tissue_number integer,
+    "sample-tissue" text,
+    cell_type_number integer,
+    "sample-cell_type" text,
+    disease_number integer,
+    "sample-disease" text,
+    "sample-description" text,
+    custom_number integer,
+    "sample-custom" text,
+    ms_run integer NOT NULL,
+    "ms_run-location" text NOT NULL,
+    "ms_run-instrument_ref" integer,
+    "ms_run-format" text,
+    "ms_run-id_format" text,
+    fragmentation_method_number integer,
+    "ms_run-fragmentation_method" text,
+    scan_polarity_number integer NOT NULL,
+    "ms_run-scan_polarity" text NOT NULL,
+    "ms_run-hash" text,
+    "ms_run-hash_method" text,
+    assay_number integer NOT NULL,
+    assay text NOT NULL,
+    "assay-custom" text,
+    "assay-external_uri" text,
+    "assay-sample_ref" text,
+    "assay-ms_run_ref" text NOT NULL,
+    study_variable_number integer NOT NULL,
+    study_variable text NOT NULL,
+    "study_variable-assay_refs" text NOT NULL,
+    "study_variable-average_function" text,
+    "study_variable-variation_function" text,
+    "study_variable-description" text NOT NULL,
+    "study_variable-factors" text,
+    "study-custom" text,
+    cv_number integer NOT NULL,
+    "cv-label" text NOT NULL,
+    "cv-full_name" text NOT NULL,
+    "cv-version" text NOT NULL,
+    "cv-uri" text NOT NULL,
+    database_number integer NOT NULL,
+    database text NOT NULL,
+    "database-prefix" text NOT NULL,
+    "database-version" text NOT NULL,
+    "database-uri" text NOT NULL,
+    derivatization_agent_number integer,
+    derivatization_agent text,
+    "small_molecule-quantification_unit" text,
+    "small_molecule-identification_reliability" integer,
+    id_confidence_measure_number integer NOT NULL,
+    id_confidence_measure text NOT NULL,
+    "colunit-small_molecule" text,
+    "colunit-small_molecule_evidence" text,
+    "mzTab-ID" text NOT NULL
 );
 
 
-ALTER TABLE public."PUBLICATIONS" OWNER TO postgres;
+ALTER TABLE public."MzTab-M_Metadata_metabolomics" OWNER TO postgres;
 
 --
--- TOC entry 212 (class 1259 OID 17526)
--- Name: PUBLICATIONS_Publication_ID_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+-- TOC entry 242 (class 1259 OID 18419)
+-- Name: MzTab-M_Small_Molecule_Evidence_(SME)_Section; Type: TABLE; Schema: public; Owner: postgres
```

Comment:
Let's also remove `-` and any parentheses from the table names (and sequence/constraint names) as well

Reply:
done